### ISSUE TITLE
Implement Functionally Equivalent Extraction Attack

### ISSUE THREAD

**Issue by beat-buesser** [IBM/adversarial-robustness-toolbox#216]
Implement Functionally Equivalent Extraction Attack for neural networks with 2 fully-connected layers with ReLU activation and logits output as described by Jagielski et al. 2019 (https://arxiv.org/abs/1909.01838):.
